packages/cli/lib/template-listings-helper.js

```diff
@@ -0,0 +1,81 @@
+const os = require('os');
+const { join } = require('path');
+const fetch = require('isomorphic-unfetch');
+
+const fs = require('./fs');
+const { info, error } = require('./util');
+
+const {
+	TEMPLATES_REPO_URL,
+	TEMPLATES_CACHE_FOLDER,
+	TEMPLATES_CACHE_FILENAME,
+	FALLBACK_TEMPLATE_OPTIONS,
+} = require('./constants.js');
+
+async function updateTemplatesCache() {
+	const cacheFilePath = join(
+		os.homedir(),
+		TEMPLATES_CACHE_FOLDER,
+		TEMPLATES_CACHE_FILENAME
+	);
+
+	try {
+		const repos = await fetch(TEMPLATES_REPO_URL).then(r => r.json());
+		await fs.writeFile(cacheFilePath, JSON.stringify(repos, null, 2), 'utf-8');
+	} catch (err) {
+		error(`\nFailed to update template cache\n ${err}`);
```

Comment:
so what happens when i am disconnected from the network?
console logs `Failed to update template cache`?